apps/designer/.env.example

```diff
@@ -2,4 +2,8 @@ DATABASE_URL=postgresql://user:pass@localhost/webstudio
 AUTH_SECRET="# Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32"
 GH_CLIENT_SECRET=
 GH_CLIENT_ID=
-DEV_LOGIN=true
\ No newline at end of file
+DEV_LOGIN=true
+
+
+# relative to the public directory
+FILE_UPLOAD_PATH="uploads"
```

Comment:
should we allow upload anywhere? without contraining to public directory? idk

Reply:
Nope, it won't work outside of public

Reply:
enforced by remix?

Reply:
yap